collections/_article/release-candidate-godot-3-5-3-rc-1.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Release candidate: Godot 3.5.3 RC 1"
+excerpt: "A new maintenance update is in the works for the 3.5 stable branch, to update Android templates to target Android 13, and fix a number of other platform porting bugs."
+categories: ["pre-release"]
+author: Rémi Verschelde
+image: /storage/blog/covers/release-candidate-godot-3-5-3-rc-1.webp
+date: 2023-09-08 11:00:00
+---
+
+It's been a while since our [last maintenance release for Godot 3](/article/maintenance-release-godot-3-5-2/)! While the development focus is on the Godot 4 branches, we're still making steady progress towards the Godot 3.6 feature release, currently in [beta phase](/article/dev-snapshot-godot-3-6-beta-3/). But we also need occasional updates to the current 3.5 branch which is used in production by many.
```

Comment:
```suggestion
It's been a while since our [lastest maintenance release for Godot 3](/article/maintenance-release-godot-3-5-2/)! While the development focus is on the Godot 4 branches, we're still making steady progress towards the Godot 3.6 feature release, currently in [beta phase](/article/dev-snapshot-godot-3-6-beta-3/). But we also need occasional updates to the current 3.5 branch which is used in production by many.
```

mb?

Reply:
Hm not sure, I get 5M results on google for "since our last", 800k for "since our latest".

Reply:
I'm just a bit concerned it can be mistaken that 3.5 is _the_ last 3.x release at a glance.

Reply:
Maybe "previous"?